pkgs/by-name/sl/slicer3d/package.nix

```diff
@@ -0,0 +1,133 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+  hwloc,
+  fetchurl,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+
+  hwloc_old = hwloc.overrideAttrs (
+    old:
+    let
+      hwloc_version = "1.11.9";
+    in
+    {
+      version = hwloc_version;
+      src = fetchurl {
+        url = "https://www.open-mpi.org/software/hwloc/v${lib.versions.majorMinor hwloc_version}/downloads/hwloc-${hwloc_version}.tar.bz2";
+        sha256 = "sha256-OUMzGEJI1jyycIqXblfwUzfQO7UMM6owl/9cWnSoUWQ=";
+      };
+    }
+  );
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "../Slicer";
```

Comment:
No relative paths — Slicer would be placed in $PATH anyway
```suggestion
    exec = "Slicer";
```

Reply:
Slicer isn't placed in $PATH since it's at the top level and not under `$out/bin`; or at least, it's not placed in path when I'm building it.

Reply:
Yeah, unless I'm doing something wrong, the top level isn't placed in `$PATH`
I also am unable to move that executable since it refers to other executable relatively. If you don't want a relative path, should I just change it back to `SlicerApp-real`?